python/grass/jupyter/timeseriesmap.py

```diff
@@ -394,14 +390,14 @@ def view_image(date):
         # Return interact widget with image and slider
         widgets.interact(view_image, date=slider)
 
-    def animate(
+    def save(
         self,
+        filename=None,
```

Comment:
None for filename does not longer make sense, no? Also the tests fails in relation to that.